content/en/blog/2023/istio-graduates-within-cncf/index.md

```diff
@@ -0,0 +1,48 @@
+
+We would like to thank our TOC sponsors [Emily Fox](https://www.cncf.io/people/technical-oversight-committee/?p=emily-fox) and [Nikhita Raghunath](https://www.cncf.io/people/technical-oversight-committee/?p=nikhita-raghunath), and everyone who has collaborated over the past six years on Istio's design, development, and deployment.
+
+As before, project work continues uninterrupted. We were excited to [bring ambient mesh to Alpha in Istio 1.18](/news/releases/1.18.x/announcing-1.18/#ambient-mesh) and are continuing to drive it to production readiness. Sidecar deployments remain the recommended method of using Istio, and our [1.19 release](https://github.com/istio/istio/wiki/Istio-Release-1.19) will support a [new sidecar container feature](https://github.com/kubernetes/kubernetes/pull/116429) in Alpha in Kubernetes 1.28.
+
+We have been delighted to welcome Microsoft to our community after [their decision to archive the Open Service Mesh project and collaborate together on Istio](https://openservicemesh.io/blog/osm-project-update/). As the [third most active CNCF project](https://all.devstats.cncf.io/d/53/projects-health-table?orgId=1) in terms of PRs, and with [support from over 20 vendors](/about/ecosystem/) and [dozens of contributing companies](https://istio.devstats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20year&var-metric=prs), there is simply no better choice for a service mesh.
+
+We would like to invite the Istio community to [submit a talk to the upcoming virtual IstioCon 2023](https://sessionize.com/istiocon-2023), the companion [full day, in-person event](https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/co-located-events/istiocon-call-for-proposals-cn/#preparing-to-submit-your-proposal-cn) co-located with KubeCon China in Shanghai, or [Istio Day](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/co-located-events/istio-day/#call-for-proposals) co-located with KubeCon NA in Chicago.
+
+---
+
+When we [announced our incubation](/blog/2023/istio-graduates-within-cncf/), we mentioned that the journey began with Istio's inception in 2016. One of the great things about collaborative open source projects is that people come and go from employers, but their affiliation with a project can remain. Some of our original contributors founded companies based on Istio; some moved to other companies that support it; and some are still working on it at Google or IBM, six years later.
+
+The announcement from the CNCF and blog posts from [Intel](https://www.intel.com/content/www/us/en/developer/articles/community/Intel-Service-Mesh-Optimizes-and-Protects-Istio-Service-Mesh), [Red Hat](https://cloud.redhat.com/blog/red-hat-congratulates-istio-on-graduating-at-the-cncf) and [Solo.io](https://www.solo.io/blog/istio-graduates-cncf) (with more to come) summarize the thoughts and feelings of those working on the project today.
```

Comment:
I expect that there will be more link to come (it even suggests that), so this will be updated a bit over the next few days. I'll plan to add a link to IBM's response here as well.